src/lib/Keys.ts

```diff
@@ -391,13 +403,25 @@ export class Ed25519 extends AsymmetricKey {
   }
 }
 
+/**
+ * Secp256k1 variant of `AsymmetricKey`
+ * @privateRemarks
+ * Needs origin notation
```

Comment:
Which libraries can be the origin notation?
- [secp256k1](https://www.npmjs.com/package/secp256k1)
- [eccrypto](https://www.npmjs.com/package/eccrypto)
- [ethereum-cryptography/secp256k1](https://www.npmjs.com/package/ethereum-cryptography#secp256k1-curve)

FYI: @hoffmannjan

Reply:
OK so we need our own notation, @gyroflaw could you prepare it and add as part of this PR?

Reply:
sure will do

Reply:
nice!